src/mpid/ch4/netmod/ofi/ofi_send.h

```diff
@@ -16,7 +16,8 @@
 
 MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_send_lightweight(const void *buf,
                                                         size_t data_sz,
-                                                        int rank,
+                                                        int src_rank,
```

Comment:
This is adding `src_rank` to the argument list for send operations where it can be gotten from `comm->rank`. I think renaming to `dst_rank` is fine. The extra argument I'm not seeing the need for strongly.

Reply:
I see what you mean now. I've taken this out.